docs/source/technical_tutorials/search/federated-collection-discovery/collection_discovery.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<p>Be more clear this is the API that runs the website described above, in case you want to interact via code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/541/discussion/#comment-2632749331'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>